runtime/compiler/env/VMJ9Server.cpp

```diff
@@ -1805,6 +1805,53 @@ TR_J9ServerVM::getInvokeExactThunkHelperAddress(TR::Compilation *comp, TR::Symbo
    return helper;
    }
 
+bool 
+TR_J9ServerVM::noMultipleConcreteClasses(List<TR_PersistentClassInfo>* subClasses)
+   {
+   TR::Compilation *comp = TR::comp();
+   int count = 0;
+   TR_ScratchList<TR_PersistentClassInfo> subClassesNotCached(comp->trMemory());
+
+   // traverse though classes that are cached on server
+   ListIterator<TR_PersistentClassInfo> i(subClasses);
+   for (TR_PersistentClassInfo *ptClassInfo = i.getFirst(); ptClassInfo; ptClassInfo = i.getNext())
+      {
+      TR_OpaqueClassBlock *clazz = ptClassInfo->getClassId();
+      J9Class *j9clazz = TR::Compiler->cls.convertClassOffsetToClassPtr(clazz);
+      auto romClass = JITServerHelpers::getRemoteROMClassIfCached(_compInfoPT->getClientData(), j9clazz);
+      if (romClass == NULL)
+         {
+         subClassesNotCached.add(ptClassInfo);
+         continue;
```

Comment:
You could eliminate this `continue` if you put everything else under the `else` part of the `if`:
```
      if (romClass == NULL)
         {
         subClassesNotCached.add(ptClassInfo);
         }
        else
          {
           ...
           }